rs/registry/canister/tests/swap_node_in_subnet_directly.rs

```diff
@@ -0,0 +1,33 @@
+use dfn_candid::candid;
+use ic_nns_test_utils::itest_helpers::{local_test_on_nns_subnet, set_up_registry_canister};
+use ic_types::PrincipalId;
+use registry_canister::{
+    init::RegistryCanisterInitPayloadBuilder,
+    mutations::do_swap_node_in_subnet_directly::{SwapError, SwapNodeInSubnetDirectlyPayload},
+};
+
+// This test ensures that we are not enabling this feature on any network until it
+// is fully implemented.
+//
+// TODO(DRE-551): adapt the logic of the test to not fail if the feature is enabled.
+#[test]
+fn ensure_feature_is_turned_off() {
+    local_test_on_nns_subnet(|runtime| async move {
```

Comment:
We are generally using state machine tests or pocket IC tests now.  nns/integration_tests has some examples, as does nervous_system/integration_tests (for pocketIC).

PocketIC is preferred.